base/some.jl

```diff
@@ -34,6 +34,9 @@ end
 
 convert(::Type{T}, x::T) where {T>:Nothing} = x
 convert(::Type{T}, x) where {T>:Nothing} = convert(nonnothingtype_checked(T), x)
+convert(::Type{Nothing}, x) = throw(MethodError(convert, (Nothing, x)))
+convert(::Type{Nothing}, x::T) where {T>:Nothing} = throw(MethodError(convert, (Nothing, x)))
```

Comment:
These two are equivalent, `methods(convert)` should only list one of them, in particular the latter as it has replaced the former. If you find that you need both to get the desired effect, something weird is going on.

Reply:
Ah yes, you are correct:
```julia
julia> methods(convert, Tuple{Type{Nothing}, Any})
# 2 methods for generic function "convert":
[1] convert(::Type{Nothing}, ::Nothing) in Base at some.jl:39
[2] convert(::Type{Nothing}, x::T) where T>:Nothing in Base at some.jl:38
```
I think I was just a bit confused because I previously assumed the definition in line 35 would just be equivalent to defining `convert(::Type{Nothing}, ::Nothing)`, but that caused problems.